compose-tests/src/androidTest/java/radiography/test/compose/ComposeUiTest.kt

```diff
@@ -597,14 +597,14 @@ class ComposeUiTest {
       """
       |CompositionLocalProvider:
       |${BLANK}CompositionLocalProvider { test-tag:"parent" }
-      |${BLANK}╰─LazyColumn { test-tag:"list", vertical-scroll-axis-range:"ScrollAxisRange(value=0.0, maxValue=Infinity)" }
+      |${BLANK}╰─LazyColumn { test-tag:"list", vertical-scroll-axis-range:"ScrollAxisRange(value=0.0, maxValue=0.0)" }
```

Comment:
What information do you see that is missing? In this diff, I really only notice the name changes and the maxValue value.

If we can not lose information, that's preferred. But I'd hard to judge that when I don't understand what we're losing.

Reply:
The type `SaveableStateProvider` was replaced with `SkippableItem` - this loses the information that there is a SaveableStateProvider wrapping the node.

The kdoc for SkippableItem says:
```
Hack around skippable functions to force skip SaveableStateProvider and Item block when nothing changed. It allows us to skip heavy-weight composition local providers.
```

So this replacement seems more like a implementation detail for optimization and isn't what we should be exposing

Reply:
Ah, yeah. Thanks for elaborating there. If we can get to that child, that's probably preferred, then

Reply:
`SaveableStateProvider` isn't inline, but I think there was an optimization to avoid making groups in many cases where they're not actually needed, and I wonder if that's why it's not showing up here.

Reply:
That makes sense. I don't know how I would change this behavior to match the previous, so I don't plan on touching this further unless someone can suggest how to do so.